2021-11-26 23:09:18,751 INFO    MainThread:4816 [wandb_setup.py:_flush():71] setting env: {'api_key': '0f579240b99e6ed291924359c5de323258326760'}
2021-11-26 23:09:18,751 INFO    MainThread:4816 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-26 23:09:18,751 INFO    MainThread:4816 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211126_230918-3uuqjzp8\logs\debug.log
2021-11-26 23:09:18,751 INFO    MainThread:4816 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211126_230918-3uuqjzp8\logs\debug-internal.log
2021-11-26 23:09:18,752 INFO    MainThread:4816 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000286E70FA130>
2021-11-26 23:09:18,754 INFO    MainThread:4816 [wandb_init.py:init():390] calling init triggers
2021-11-26 23:09:18,754 INFO    MainThread:4816 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-26 23:09:18,754 INFO    MainThread:4816 [wandb_init.py:init():435] starting backend
2021-11-26 23:09:18,754 INFO    MainThread:4816 [backend.py:_multiprocessing_setup():94] multiprocessing start_methods=spawn, using: spawn
2021-11-26 23:09:18,757 INFO    MainThread:4816 [backend.py:ensure_launched():198] starting backend process...
2021-11-26 23:09:18,827 INFO    MainThread:4816 [backend.py:ensure_launched():203] started backend process with pid: 11444
2021-11-26 23:09:18,827 INFO    MainThread:4816 [wandb_init.py:init():444] backend started and connected
2021-11-26 23:09:18,839 INFO    MainThread:4816 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-26 23:09:18,839 INFO    MainThread:4816 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-26 23:09:18,839 INFO    MainThread:4816 [wandb_init.py:init():503] updated telemetry
2021-11-26 23:09:18,871 INFO    MainThread:4816 [wandb_init.py:init():533] communicating current version
2021-11-26 23:09:20,128 INFO    MainThread:4816 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-26 23:09:20,129 INFO    MainThread:4816 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-26 23:09:20,470 INFO    MainThread:4816 [wandb_init.py:init():576] starting run threads in backend
2021-11-26 23:09:21,518 INFO    MainThread:4816 [wandb_run.py:_console_start():1693] atexit reg
2021-11-26 23:09:21,519 INFO    MainThread:4816 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-26 23:09:21,519 INFO    MainThread:4816 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-26 23:09:21,523 INFO    MainThread:4816 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-26 23:09:21,523 INFO    MainThread:4816 [wandb_init.py:init():603] run started, returning control to user process
2021-11-26 23:09:21,744 INFO    MainThread:4816 [wandb_run.py:_config_callback():911] config_cb ('_wandb', 'viz', 'feature_importances') {'id': 'wandb/feature_importances/v1', 'historyFieldSettings': {'key': 'feature_importances', 'x-axis': '_step'}} None
2021-11-26 23:09:22,674 INFO    MainThread:4816 [wandb_run.py:_config_callback():911] config_cb ('_wandb', 'viz', 'confusion_matrix') {'id': 'wandb/confusion_matrix/v1', 'historyFieldSettings': {'key': 'confusion_matrix', 'x-axis': '_step'}} None
2021-11-26 23:09:24,307 INFO    MainThread:4816 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-26 23:09:24,307 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:10:34,239 INFO    MainThread:4816 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-26 23:10:34,240 INFO    MainThread:4816 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-26 23:10:34,240 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:10:38,738 INFO    MainThread:4816 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-26 23:10:38,739 INFO    MainThread:4816 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-26 23:10:38,739 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:10:39,953 INFO    MainThread:4816 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-26 23:10:39,982 INFO    MainThread:4816 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-26 23:10:39,982 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:11:10,952 INFO    MainThread:4816 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-26 23:11:11,390 INFO    MainThread:4816 [wandb_setup.py:_flush():71] setting env: {'api_key': '0f579240b99e6ed291924359c5de323258326760'}
2021-11-26 23:11:11,390 INFO    MainThread:4816 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-26 23:11:11,390 INFO    MainThread:4816 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211126_231111-3vr7vuc8\logs\debug.log
2021-11-26 23:11:11,390 INFO    MainThread:4816 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211126_231111-3vr7vuc8\logs\debug-internal.log
2021-11-26 23:11:11,390 INFO    MainThread:4816 [wandb_init.py:init():390] calling init triggers
2021-11-26 23:11:11,391 INFO    MainThread:4816 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-26 23:11:11,391 INFO    MainThread:4816 [wandb_init.py:init():410] re-initializing run, found existing run on stack: 3uuqjzp8
2021-11-26 23:11:11,392 INFO    MainThread:4816 [wandb_run.py:finish():1347] finishing run rahav-manoharan/demo/3uuqjzp8
2021-11-26 23:11:11,393 INFO    MainThread:4816 [jupyter.py:save_history():429] not saving jupyter history
2021-11-26 23:11:11,393 INFO    MainThread:4816 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-26 23:11:11,393 INFO    MainThread:4816 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-11-26 23:11:11,393 INFO    MainThread:4816 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-26 23:11:11,393 INFO    MainThread:4816 [wandb_run.py:_restore():1635] restore
2021-11-26 23:11:11,854 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 3489
}

2021-11-26 23:11:12,218 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 3489
}

2021-11-26 23:11:12,437 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 9145
}

2021-11-26 23:11:12,543 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 9145
}

2021-11-26 23:11:12,657 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 9145
}

2021-11-26 23:11:12,764 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 9145
}

2021-11-26 23:11:12,871 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 9145
}

2021-11-26 23:11:12,981 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 9145
}

2021-11-26 23:11:13,090 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 3489
  total_bytes: 9145
}

2021-11-26 23:11:13,199 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 9145
  total_bytes: 9145
}

2021-11-26 23:11:13,310 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 9145
  total_bytes: 9145
}

2021-11-26 23:11:13,422 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 9145
  total_bytes: 9145
}

2021-11-26 23:11:13,532 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 9145
  total_bytes: 9145
}

2021-11-26 23:11:13,640 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 9145
  total_bytes: 9145
}

2021-11-26 23:11:13,749 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 9145
  total_bytes: 9145
}

2021-11-26 23:11:14,379 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 9145
  total_bytes: 9145
}

2021-11-26 23:11:14,779 INFO    MainThread:4816 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 2
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 9145
  total_bytes: 9145
}
local_info {
}

2021-11-26 23:11:15,868 INFO    MainThread:4816 [wandb_run.py:_append_history():2011] rendering history
2021-11-26 23:11:15,869 INFO    MainThread:4816 [wandb_run.py:_append_summary():1966] rendering summary
2021-11-26 23:11:15,869 INFO    MainThread:4816 [wandb_run.py:_append_files():2061] logging synced files
